doc/Language/exceptions.rakudoc

```diff
@@ -380,6 +380,34 @@ C<throw> can be viewed as the method form of C<die>, just that in this
 particular case, the sub and method forms of the routine have different
 names.
 
+=head1 Delaying exceptions as failures
```

Comment:
`fail`ing can be described as delaying exceptions. From that perspective the current title reads as "Delaying exceptions as delaying exceptions".

What about just "Delaying exceptions"? Then the reader will be predisposed to understand that `fail` and "failure" are the chosen concrete terms for the abstract notion of "delaying exceptions".

Then again, failures aren't really "delaying exceptions" in the sense that, if producing code produces them, the whole idea is for consuming code to handle them properly so they do _not_ throw -- and as such, they're not delayed exceptions, but instead just regular error handling like it all used to be in the old days.

Perhaps the title could be "Converting exceptions to error values"?